getting-started.md

```diff
@@ -48,43 +46,51 @@ Once it is up and running you can get a shell inside the container with:
 See [Managing containers](https://discourse.ubuntu.com/t/managing-containers/17763) for more details.
 
 ## Create an application
-**AMS** provides functionality to manage Android applications for you. To let **AMS** manage your application, you need the APK of the application and a `manifest.yaml` which looks like this in its simplest form:
+
+You can create applications through the web-based dashboard or on the command line with AMS.
+
+Both alternatives require you to provide the APK of the Android application, its name and its [instance type](https://discourse.ubuntu.com/t/instances-types-reference/17764).
+
+> **Hint**: If you deployed Anbox Cloud with the streaming stack, use an instance type with GPU support like `g2.3`. Otherwise the container will not get access to a GPU for rendering and video encoding.
```

Comment:
This is misleading, you can deploy the streaming stack on a system without a GPU, which means that an application created with a `g*` instance-type won't run because of the lack of GPU.
Every instance-type uses a GPU if available and software encoder otherwise, `g*` instances **require** a GPU